CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1 @@
+Please see [our code of conduct policy](http://docs.librenms.org/General/CODE_OF_CONDUCT/).
```

Comment:
Do you mean because of the change I did in mkdocs? If not, then it already exists.